app/ledger/n3NeonLedger.js

```diff
@@ -14,6 +22,32 @@ export const MESSAGES = {
     'Error parsing transaction. Make sure your NEO3 Ledger app version is up to date',
 }
 
+/**
+ * Evaluates Transport Error thrown and rewrite the error message to be more user friendly.
+ * @param {Error} err
+ * @return {Error}
+ */
+const evalTransportError = error => {
+  const err = cloneDeep(error)
```

Comment:
I wonder if this cloneDeep has anything todo with this version working and the one in `neon-js` not because that's arguably the only difference besides some invalid status words.